### DISCUSSION TITLE
How can I set specific host for advertised.listener with nodeport type?

### DISCUSSION THREAD

**Discussion by arezvani** [strimzi/strimzi-kafka-operator#8132] [Q&A]
Hi,
I want to have external access to my kafka cluster. I use nodeport type for listener but I didn't find any option that I can set my desired host. I have a load balancer for worker nodes that have a public IP and i set a host and domain for that for example ali.kafka.com.

What should I do? Please help me

**Comment by scholzj** [comment]
You mean that you want to just override the advertised host with your own DNS name of your load balancer? Please look in the docs for `advertisedHost`: https://strimzi.io/docs/operators/latest/full/configuring.html#type-GenericKafkaListenerConfigurationBroker-reference
